server/models/transactions.js

```diff
@@ -1,10 +1,29 @@
 // imports goes here
-class Transaction {
+import uuidv4 from 'uuid/v4';
+import moment from 'moment';
+import db from '../config/db';
 
-  
-    async Transaction(data) {
-        console.log('Transaction');
+class Transaction {
+  async saveMoney(data) {
+    this.savedMoney = [
+      uuidv4(),
+      data.amount,
+      // uuidv4(),
+      moment(new Date()),
+      data.transactionCode,
+      data.comment,
+    ];
+    try {
+      const queryMoneySaved = `INSERT INTO 
```

Comment:
Would you move this query to the queries file under config folder for better code structure?